target/create.ps1

```diff
@@ -37,7 +37,11 @@ function Resolve-{connectorName}Error {
             # $httpErrorObj.FriendlyMessage = $errorDetailsObject.message
             $httpErrorObj.FriendlyMessage = $httpErrorObj.ErrorDetails # Temporarily assignment
         } catch {
-            $httpErrorObj.FriendlyMessage = $httpErrorObj.ErrorDetails
+            if ($_.Exception.Message) {
```

Comment:
De if/else zouden naar mijn mening weg mogen. Immers, als we in de `catch` zitten dan hebben we sowieso een exception. En de kans is dan ook zeer aannemlijk dat we een `Exception.Message` hebben. Want; daar zorgt .NET gewoon voor. De `ConvertFrom-Json` staat los van de API call. 

De `catch` zou dan worden:

```powershell
 $httpErrorObj.FriendlyMessage = "Error: [$($httpErrorObj.ErrorDetails)] [$($_.Exception.Message)]"
 ```

Dan heb je alsnog de `$httpErrorObj.ErrorDetails`. Ofwel door regel 25. Danwel door regel 30. (In ruwe vorm). Plus; de exception message omdat de convert niet goed ging op regel 35.

Reply:
Op mijn demo API ziet het object er dan als volgt uit:

```powershell

ScriptLineNumber =
75
Line =
"    Invoke-RestMethod -Uri "$baseUrl/api/user/search?email=ac.doe@example" -Method 'GET' -Headers $headers
"
ErrorDetails =
"Response status code does not indicate success: 404 (Not Found)."
FriendlyMessage =
"Error: [Response status code does not indicate success: 404 (Not Found).] [Conversion from JSON failed with error: Unexpected character encountered while parsing value: R. Path '', line 0, position 0.]"
```